### PR TITLE
Hotfix/fix integration tests

### PR DESCRIPTION
**Pull request by hjoaquim**
This PR aims to fix the broken integration tests.
Mostly, due to unhandled empty results when accessing a function from a source with an undefined API key.

To run the integration tests:

`python terminal.py -t`

### CODE REVIEW

File: openbb_terminal/cryptocurrency/onchain/shroom_model.py [hunk 1]
```diff
@@ -32,9 +33,10 @@ def create_query(query: str):
         },
     )
     if r.status_code != 200:
-        raise Exception(
-            f"Error creating query, got response: {r.text} with status code: {str(r.status_code)}"
+        console.print(
+            f"[red]Error creating query, got response: {r.text} with status code: {str(r.status_code)}"
```
Comment (jmaslek) [inline openbb_terminal/cryptocurrency/onchain/shroom_model.py:37]: WIll need a [/red]

File: openbb_terminal/cryptocurrency/onchain/shroom_model.py [hunk 2]
```diff
@@ -49,9 +51,10 @@ def get_query_results(token):
         },
     )
     if r.status_code != 200:
-        raise Exception(
-            f"Error creating query, got response: {r.text} with status code: {str(r.status_code)}"
+        console.print(
```
Comment (jmaslek) [inline openbb_terminal/cryptocurrency/onchain/shroom_model.py:54]: same

File: openbb_terminal/cryptocurrency/onchain/shroom_model.py [hunk 3]
```diff
@@ -188,7 +197,8 @@ def get_total_value_locked(
     """
 
     if not (user_address or address_name):
-        raise Exception("No user address or address name provided")
+        console.print("[red]No user address or address name provided.")
```
Comment (jmaslek) [inline openbb_terminal/cryptocurrency/onchain/shroom_model.py:200]: *

File: openbb_terminal/cryptocurrency/onchain/shroom_view.py [hunk 4]
```diff
@@ -46,6 +46,7 @@ def display_daily_transactions(
     symbols = ["DAI", "USDT", "BUSD", "USDC"]
     df = get_daily_transactions(symbols)
     if df.empty:
+        console.print("[red]No data found.")
```
Comment (jmaslek) [inline openbb_terminal/cryptocurrency/onchain/shroom_view.py:49]: *

File: openbb_terminal/cryptocurrency/onchain/shroom_view.py [hunk 5]
```diff
@@ -103,7 +104,7 @@ def display_dapp_stats(
     """
     df = get_dapp_stats(platform=platform)
     if df.empty:
-        console.print("No data found.", "\n")
+        console.print("[red]No data found.")
```
Comment (jmaslek) [inline openbb_terminal/cryptocurrency/onchain/shroom_view.py:107]: *

File: openbb_terminal/cryptocurrency/onchain/shroom_view.py [hunk 6]
```diff
@@ -188,7 +189,7 @@ def display_total_value_locked(
     )
 
     if df.empty:
-        console.print("No data found.", "\n")
+        console.print("[red]No data found.")
```
Comment (jmaslek) [inline openbb_terminal/cryptocurrency/onchain/shroom_view.py:192]: *

**Comment by hjoaquim** [comment]
^
Sorry about that, thought that `[red]` in the beginning was enough, will fix it!

**Review by jmaslek — APPROVED** [review]

**Pull request merged**